pkg/utils/project/project.go

```diff
@@ -18,4 +18,4 @@ package project
 
 // Version is the karpenter app version injected during compilation
 // when using the Makefile
-var Version = "unspecified"
+var Version = "v0.7.6"
```

Comment:
IMO this shouldnt be hardcoded it should be derived from tags.

See https://github.com/Azure/azure-service-operator/blob/main/v2/internal/version/version.go#L18
https://github.com/Azure/azure-service-operator/blob/main/Taskfile.yml#L40
https://github.com/Azure/azure-service-operator/blob/main/Taskfile.yml#L33

or
https://github.com/kro-run/kro/blob/main/Makefile#L26

Reply:
It is already derived from tags/commits in our makefile. This Version needs to be removed, and user agent should be using the one from operator.